api/v1alpha3/labels.go

```diff
@@ -108,6 +110,9 @@ const (
 
 	// APIServerRoleTagValue describes the value for the apiserver role.
 	APIServerRoleTagValue = "apiserver"
+
+	// NameKubernetesObjectUID is the tag name for the UID of the "creating" kubernetes object.
+	NameKubernetesObjectUID = "k8s-io-owner-uid"
```

Comment:
@justinsb I'd warn against using UID here, since the UID would change if the k8s resource is moved to a different Cluster API management cluster or if it is restored from a backup.